flake.nix

```diff
@@ -62,7 +62,7 @@
                 name = "topiary-inplace";
                 text = ''
                   for FILE; do
-                    if ${code.topiary-cli}/bin/topiary --in-place --input-file "$FILE"; then
+                    if ${code.topiary-cli}/bin/topiary --in-place --input-files "$FILE"; then
```

Comment:
You don't need this loop any more. I think you can replace the call to Topiary with:

```bash
topiary --in-place --input-files "$@"
```